src/wrappers/themis/react-native-themis/src/index.js

```diff
@@ -44,16 +56,8 @@ export function symmetricKey64() {
 }
 ;
 export function secureCellSealWithSymmetricKeyEncrypt64(symmetricKey64, plaintext, context = "") {
-    if (plaintext === "" || plaintext === undefined || plaintext === null) {
-        throw new Error("Parameter plaintext can not be empty");
-    }
-    if (symmetricKey64 === "" || symmetricKey64 === undefined || symmetricKey64 === null) {
-        throw new Error("Parameter symmetricKey64 can not be empty");
-    }
-    if (!isBase64(symmetricKey64)) {
-        throw new Error("Parameter symmetricKey64 is not base64 encoded");
-    }
-    const symmetricKey = Array.from(Buffer.from(symmetricKey64, 'base64'));
+    convertInputBase64(plaintext, "plaintext", false); // check plaintext is not empty
+    const symmetricKey = convertInputBase64(symmetricKey64, "symmetricKey64", true); // check symmetricKey64 is not empty and base64 encoded
```

Comment:
I think `context` is missing from all the checks I see. Is this intentional?

I believe it should be okay to have `undefined` (not provided on call), `null` (explicitly omitted), `""` (ditto), but shouldn't it be base64-encoded like everything else? In context imprint mode it *is* converted.